### PR TITLE
Adding redirects as needed. Duplicates removed.

### PR DESCRIPTION
**Pull request by SgtCoin**
Adding in redirects as directed by external SEO team. Duplicates removed, requesting review.

### CODE REVIEW

File: .gitbook.yaml [hunk 1]
```diff
@@ -388,3 +388,28 @@ redirects:
...
+  get-started/go-filecoin-tutorial/Storing-on-Filecoin.html:	storage-providers/basics/quickstart-guide.md
+  getstarted/lotus/switing-networks:	basics/what-is-filecoin/networks.md
+  go-filecoin-tutorial/Explorer-and-Dashboard.html:	reference/general.md
+  go-filecoin-tutorial/Getting-Started.html:	storage-providers/basics/quickstart-guide.md
+  go-filecoin-tutorial/How-Filecoin-Works.html:	basics/what-is-filecoin.md
+  storage-providers/filecoin-economics/block-rewards: basics/what-is-filecoin.md
+  images/social-card.png: basics/what-is-filecoin.md
+  mine/mining-rewards.html:	storage-providers/filecoin-economics/block-rewards.md
+  smart-contracts/developing-contracts/get-test-tokens/faucet-get-address_hu511fb870b227f441051bc067b9f419d0_204132_441x0_resize_q75_h2_box_3.webp:	networks/calibration.md
+  smart-contracts/developing-contracts/get-test-tokens/faucet-verify_huf20c2a4bf454bdefc9cb52bafd199d7f_1516790_1440x0_resize_q75_h2_box_3.webp	smart-contracts/fundamentals/the-fvm: basics/what-is-filecoin.md
```
Comment (johnnymatthews) [inline .gitbook.yaml:413]: Why are there redirects for `.webp` files?

Comment (johnnymatthews) [inline .gitbook.yaml:413]: Why are there redirects for `webp` files?

Comment (johnnymatthews) [reply .gitbook.yaml:413]: I have a feeling that they'll cause errors with GitBook. So yeah, please remove.

Comment (SgtCoin) [reply .gitbook.yaml]: Roger that, I removed all the png redirects.

**Review by johnnymatthews — APPROVED** [review]
Looks good!

**Pull request merged**